base/operators.jl

```diff
@@ -341,6 +341,16 @@ false
 julia> a === a
 true
 ```
+
+# Extended help
+It might be possible to construct a program that is actually able to distinguish
+two egal values. For example, one could heap-allocate two integers,
```

Comment:
> that is actually able to distinguish two egal values

I think this is not an idea formulation because the program doesn't actually distinguish two egal values.